src/main/java/views/add_edit_category_views/EditCategoryV.java

```diff
@@ -97,16 +98,11 @@ public void actionPerformed(ActionEvent evt) {
             else if (budgetInput.getText().isEmpty()){
                 JOptionPane.showMessageDialog(this,"Please enter the previous category budget if you don't wish to edit. Thanks.");
             }
-            // Check if user selects an old category.
-            else if (categoryCombo.getSelectedItem() == null) {
-                JOptionPane.showMessageDialog( this, "Please select a category to edit.");
-            }
             else {
                 tryUseCaseEdit();
             }
```

Comment:
This is just kind of a nitpick since it doesn't really affect how the code works, but why are these ifs embedded inside the else and not just continuing the if-else chain?

Reply:
It's because there wasn't the else if before and I didn't change the rest of my code lol. If you think that affects code reading, I can fix it! lmk